content/app/content_main_runner_impl.cc

```diff
@@ -675,6 +675,8 @@ int NO_STACK_PROTECTOR RunZygote(ContentMainDelegate* delegate) {
       return kMainFunctions[i].function(std::move(main_params));
   }
 
+  content::RenderFrameHost::AllowInjectingJavaScript();
```

Comment:
Given that this is in common / shared code, and modifies Chrome default behavior, we should have a conditional here.

Either
- runtime check 
- compile time check

@andrewsavage1 what's the latest with this - how should we feature-gate this ?

Reply:
Use `BUILDFLAG(IS_COBALT)` from `//build/buildflag.h`

Reply:
Id' expect this to be called from a cobalt/...browser_main_parts.cc file, like https://source.chromium.org/chromium/chromium/src/+/main:chromecast/browser/cast_browser_main_parts.cc;l=691?q=AllowInjectingJavaScript

Reply:
There's some info on the browser parts here : https://source.chromium.org/chromium/chromium/src/+/main:ios/web/public/init/web_main_parts.h;l=10-57?q=PreMainMessageLoopRun

Android Webview does that too, here: https://source.chromium.org/chromium/chromium/src/+/main:android_webview/browser/aw_browser_main_parts.cc;l=322?q=AllowInjectingJavaScript

But in any case, I'd expect this to be called from some place in //cobalt/browser. Could that be done?

Reply:
moved into cobalt_main_delegate.cc